test/e2e/metamask-ui.spec.js

```diff
@@ -1374,7 +1374,7 @@ describe('MetaMask', function () {
       await driver.delay(regularDelayMs)
 
       await driver.switchToWindow(dapp)
-      await driver.delay(tinyDelayMs)
+      await driver.delay(largeDelayMs * 2)
```

Comment:
why were the delays increased. what affected the timing?

Reply:
This might actually be out of date. Previously this PR used a different version of the test dapp. I'm going to try it without the increased delays.